server/src/configs/default.json

```diff
@@ -595,8 +595,9 @@
   },
   "scanner": {
     "backendConfig": {
-      "platform": "rdm/mad/custom",
+      "platform": "rdm/mad/dragonite/custom",
       "apiEndpoint": "http://ip:port/api/",
+      "apiQueueEndpoint": "http://ip:port/api/",
```

Comment:
Is this necessary? You can't just add `queue` to the end like we've done in the past?